src/Command/Project/ProjectSshConfigCommand.php

```diff
@@ -0,0 +1,117 @@
+<?php
+namespace Platformsh\Cli\Command\Project;
+
+use Platformsh\Cli\Command\CommandBase;
+use Platformsh\Cli\Console\AdaptiveTableCell;
```

Comment:
There are 4 unused imports here which can be removed: AdaptiveTableCell, Table, Project, and Yaml